docs/how-to-backup-your-Hubs-instance.md

```diff
@@ -1,132 +1,70 @@
-* Daily backups - kept for 7 days $7.20/month (30% of cost of droplet)
+   `npm run backup`
 
-Snapshots are a one-time save on demand.
+   a. In case you see Error: Cannot find module 'yaml', you need to do a clean install like you did in the [Beginner’s Guide, Step 12i](https://docs.hubsfoundation.org/beginners-guide-to-CE.html#12-generate-your-config-file). 
 
-* Snapshots - rate $0.06 per GiB per month, you do these on demand, they are not automatic
+   ![Capture of VS Code, Terminal window, npm run backup returning an error of Cannot find module 'yaml'](img/backups/image2.png)
 
-Note: these prices are only quotes as of November 2024. With all of these DO quotes, the costs are dependent on either what droplet costs you are paying or how much you have saved in memory (gigabytes).
+   b. In the community-edition folder, you should see a folder called data_backups. That is where the backups are stored. There is one folder per backup. The backup is named data_backup with a time stamp which is a long number representing the date as time in seconds; the bigger the number, the later in time the backup.
 
-At DO, both Backups and Snapshots save ONE backup of everything within DO’s own computers. What you pay for is either the frequency of automatic saving (for Backups) or the file space (for Snapshots).
+   ![Capture of VS Code, File Explorer, data backups folder, data backup\_175391XXXXX file](img/backups/image3.png)
+     
+   In case you are confused, in the File Explorer, select the file using the secondary mouse button (right click) and choose Reveal. The modified date field should reflect the date when the backup was last modified.
 
-## Which should I pick, [Backups](#weekly-backups) or [Snapshots](#snapshots)?
+3. In our testing, this took five to 15 minutes, but if you have a lot of data or a slow Internet connection, it could take hours. When it is done, the terminal will be back in the state where it is ready to accept a command.
 
-Backups make sense if you are a medium-to-heavy user of Hubs or if your use is critical.  For example, if you are a college professor and you have 25 student projects coming due all at once through your Hubs, you might want Daily Backups.  If you are a solo user, but your project has high value (like an art showing or a work meeting) you might want Daily or Weekly Backups.
 
-Snapshots make more sense if you are a light user of Hubs. If you can remember to save and put a copy away and if it doesn’t bother you to lose some of your Hubs work in-between when you remember to save, snapshots might be better for you.
+## **How to restore a backup**
 
-> 💡 Tip: Snapshots do backup volumes. Automated backups do not. These are not persistent volumes; these are extra volumes for off node storage that a user might purchase.
+1. Open VS Code. Be sure to be in the community-edition folder. These instructions will, by default, restore the latest or most recent backup.
```

Comment:
Should we link to the Beginner's Guide here (when talking about being in the community-edition folder), the same as we do up above (or reference the link above)?